### PR TITLE
composer: allow PHP 7 in safer way

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,10 +16,10 @@
         }
     ],
     "require": {
-        "php" : ">=5.3.0"
+        "php" : "^5.3|^7.0"
     },
     "require-dev": {
-        "phpunit/phpunit" : "4.*",
+        "phpunit/phpunit" : "^4.0|^5.0",
```
Comment (shadowhand) [inline composer.json]: Why not `4.0 - 5.0`?

Comment (TomasVotruba) [inline composer.json]: E.g. to allow 5.1

Comment (shadowhand) [inline composer.json]: Ok how about `4 - 5`?

Comment (TomasVotruba) [inline composer.json]: I understand it does the same, yet it's [not standard](https://github.com/thephpleague/glide/blob/master/composer.json#L26) for league packages.

Also it's easier to bump version, e.g to 4.7 (due to prophecy integration) in the future.

Could you link me to some repos, where do you find these?

Comment (shadowhand) [inline composer.json]: I haven't found them anywhere, it just seems cleaner than "or" versions.

Comment (TomasVotruba) [inline composer.json]: I see. I prefer these.

**Comment by TomasVotruba** [comment]
What needs to be done here to finish this?

**Comment by philsturgeon** [comment]
This is actually done in master! Sorry it never got merged, this one esacped us.

**Comment by TomasVotruba** [comment]
Awesome, thanks @philsturgeon for recent activity. It's great to see this!

**Comment by philsturgeon** [comment]
@TomasVotruba thank you :) Skeleton occasionally gets somewhat overlooked. Hopefully we'll be more on top of things now the backlog has shranked.

**Comment by TomasVotruba** [comment]
Backlog is ofter the scary part, good point.
@philsturgeon you make good work, bro, I'm happy to promote this project

**Pull request closed without merging**